gammapy/makers/background/tests/test_fov.py

```diff
@@ -326,3 +326,36 @@ def test_fov_background_maker_str():
     exclusion_mask = Map.create(binsz=0.2, width=(2, 2))
     maker_fov = FoVBackgroundMaker(exclusion_mask=exclusion_mask)
     assert "FoVBackgroundMaker" in str(maker_fov)
+
+
+@requires_data()
+def test_fov_backgr_maker_spatial_model(obs_dataset, exclusion_mask):
+    # test definition of FoVBackgroundMaker with spatial model
+    bkg_spatial_model = ConstantSpatialModel()
+
+    fov_bkg_maker = FoVBackgroundMaker(
+        method="fit",
+        exclusion_mask=exclusion_mask,
+        spectral_model="pl-norm",
+        spatial_model=bkg_spatial_model,
+    )
+    fov_bkg_maker2 = FoVBackgroundMaker(
+        method="fit",
+        exclusion_mask=exclusion_mask,
+        spectral_model="pl-norm",
+        spatial_model="const",
+    )
+
+    test_dataset = obs_dataset.copy(name="test-fov")
+
+    dataset = fov_bkg_maker.run(test_dataset)
+    dataset2 = fov_bkg_maker2.run(test_dataset)
+
+    bkg_model_spec = dataset.models[f"{dataset.name}-bkg"].spectral_model
+    bkg_model_spec2 = dataset2.models[f"{dataset2.name}-bkg"].spectral_model
+    assert not bkg_model_spec.norm.frozen
+    assert not bkg_model_spec2.norm.frozen
+    assert_allclose(bkg_model_spec.norm.value, 0.830779, rtol=1e-4)
+    assert_allclose(bkg_model_spec.tilt.value, 0.0, rtol=1e-4)
```

Comment:
I guess its not necessary to check the spectral tilt here. It is frozen by default